src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_DISCOUNT = 50;
+  const SHORT_DISCOUNT = 20;
+
+  if (days >= 7) {
+    return days * PRICE_PER_DAY - LONG_DISCOUNT;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
no need to add additional check for 7 days because you will return the value on line 36
```suggestion
  if (days >= 3) {
```